Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -106,42 +106,56 @@ class ContentBlockerHelper {
         return EnabledState(rawValue: pref) ?? .onInPrivateBrowsing
     }
 
-    fileprivate func addActiveRulesToTab() {
-        guard let ruleStore = ruleStore else { return }
+    func addActiveRulesToTab() -> Deferred<[String]> {
+        let result = Deferred<[String]>()
+        guard let ruleStore = ruleStore else { return result }
```

Comment:
Looking at the definition of ruleStore I dont think it can be nil. `default` is `Self!`

Reply:
This was failable, did it change maybe? That is handy if I don't need to treat it as optional.

Reply:
Nice catch! i can cleanup that optional rulestore stuff, API for initialization is now:
```
@available(iOS 11.0, *)
open class WKContentRuleListStore : NSObject {
    open class func `default`() -> Self!
    public convenience init!(url: URL!)
...
```